compiler/circle-input-names/src/circle-input-names.test.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <vector>
+#include <regex>
+
+class circle_input_names_test : public ::testing::Test
+{
+protected:
+  void SetUp(void) override;
+  std::vector<std::string> get_input_names_of(std::string op);
+
+protected:
+  std::string _input_names_dict_str;
+};
+
+void circle_input_names_test::SetUp(void)
+{
+  std::string cmd = std::getenv("CIRCLE_INPUT_NAMES_PATH");
+  if (cmd.empty())
+  {
+    std::runtime_error("CIRCLE_INPUT_NAMES_PATH is not found");
+    return;
+  }
+
+  FILE *fp = popen(cmd.c_str(), "r");
+  if (!fp)
+  {
+    std::runtime_error("popen() failed");
+    return;
+  }
+
+  char buff[1024];
```

Comment:
`1024` fixed length may have potential problem when we long file in the future.
maybe use string. maybe use c++ file instead of `FILE` ?

Reply:
This temporary local buffer is reused in following line if the file size exceeds the buffer size.
Anyway, I'll find a other way instead of using fixed size of buffer.

Reply:
I think current code is OK. You may revise if you like :)

Reply:
I've tried to find replacible method, but I failed.
It seems that there's no generic way to read file from `FILE *` to streambuf class.
I just left it as it is for now because current code is not hamful and follows generic use pattern. 
I'll revise if I find a better way.